app/scss/layout/_content.scss

```diff
@@ -3,8 +3,11 @@
   width: 100%;
   max-width: $site-width;
   margin: 0 auto;
-  padding: 20px;
+  padding: 80px;
+  padding-top: 25px;
   @include breakpoint($breakpoint-tablet-portrait){
-    padding: 40px;
+    padding: 80px;
```

Comment:
@bursteinz if you were trying to reduce the size of the content here all you needed to do was adjust the $site-width variable. I'm testing for all devices and your change here breaks the look on mobile so I'm returning it to what it was